### PR TITLE
IDE-864 Password Expiration window pops up for NEVER password expiration settings

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@RussWhitehead for review

**Comment by dehilsterlexis** [comment]
@RussWhitehead hold off. Got to take care of -1 correctly as well.

**Comment by dehilsterlexis** [comment]
@RussWhitehead for review

File: wlib/LoginDlg.cpp [hunk 1]
```diff
@@ -959,9 +959,12 @@ const TCHAR * GetAboutVersion(std::_tstring &version)
 
 bool CheckPasswordExpiration(int passwordExpirationWarningDays, int passwordDaysRemaining, CString &retMsg)
 {
-	if (passwordExpirationWarningDays && passwordDaysRemaining <= passwordExpirationWarningDays) {
+	if (passwordExpirationWarningDays && passwordDaysRemaining >= -1 && passwordDaysRemaining <= passwordExpirationWarningDays) {
 		CString msg;
-		msg.Format(_T("Your password expires in %i day(s)\nDo you want to change it?"), passwordDaysRemaining, retMsg);
+		if (passwordDaysRemaining == -1)
+			msg = _T("Your password has expired.\nDo you want to change it?");
```
Comment (GordonSmith) [inline wlib/LoginDlg.cpp]: If they say "no" to this, should the IDE exit?  If so you would need to be very carful that this code is correct for non PW enabled systems...

**Comment by richardkchapman** [comment]
Why is this targeting 7.2.8 (which is already Gold)? Target 7.2.x unless you are fixing a regression or build break in a specific point release

**Comment by richardkchapman** [comment]
@GordonSmith Please review

**Pull request merged**